aodn_cloud_optimised/bin/generic_cloud_optimised_creation.py

```diff
@@ -49,131 +27,236 @@
 from aodn_cloud_optimised.lib.s3Tools import s3_ls
 
```

Comment:
For all properties of BaseModel inheriting classes (models), utilise the Field method at bare minimum to supply a description for users e.g.

```suggestion
class ClusterConfig(BaseModel):
    mode: Literal['x', 'y'] | None = Field(default=None, description="The mode for the cluster, must be one of {'x', 'y'}"
```